sklearn/gaussian_process/kernels.py

```diff
@@ -685,10 +685,25 @@ def requires_vector_input(self):
 
 
 class Sum(KernelOperator):
-    """Sum-kernel k1 + k2 of two kernels k1 and k2.
+    """The `Sum` kernel takes two kernels :math:`k_1` and :math:`k_2`
+    and combines them via
 
-    The resulting kernel is defined as
-    k_sum(X, Y) = k1(X, Y) + k2(X, Y)
+    .. math::
+        k_{sum}(X, Y) = k_1(X, Y) + k_2(X, Y)
+
+    Using the Sum-kernel is equivalent to adding two kernels::
+
+
+        RBF() + RBF()
+
+    is the same as::
+
+        Sum(RBF(), RBF())
+
+    As a reference on how to best combine different kernels,
+    please refer to [1]_.
```

Comment:
This is a very useful reference, though I think it'd be more appropriate in the UG, at the end of the first paragraph of "Kernels for Gaussian Processes"